src/syncData.ts

```diff
@@ -132,6 +136,7 @@ export const initSync = (
       }
     } catch (e) {
       emit('error', { error: e, changeStream: true })
```

Comment:
Just checking my understanding: in the event of an error, we'll emit an `error` event here in mongo2crate (potentially multiple times, once for each retry), then a `processError` event in mongochangestream.

Reply:
Great thoughts.